src/rpc/server.zig

```diff
@@ -0,0 +1,554 @@
+const std = @import("std");
+const sig = @import("../sig.zig");
+
+const SnapshotGenerationInfo = sig.accounts_db.AccountsDB.SnapshotGenerationInfo;
+const FullSnapshotFileInfo = sig.accounts_db.snapshots.FullSnapshotFileInfo;
+const IncrementalSnapshotFileInfo = sig.accounts_db.snapshots.IncrementalSnapshotFileInfo;
+const ThreadPool = sig.sync.ThreadPool;
+
+const logger_scope = "rpc.Server";
```

Comment:
NIT: I'd prefer constant values defined after type aliases

Reply:
would also be good to keep global constants with SCREAMING_CASE

Reply:
e8e8dbb36634fffa7f3cc1c1a2a3d3f856459ab7

Reply:
@0xNineteen are you referring to something other than this `logger_scope` constant?

Reply:
nope - lgtm